src/3d/qgs3daxis.h

```diff
@@ -84,6 +81,11 @@ class _3D_EXPORT Qgs3DAxis : public QObject
      */
     void onViewportScaleFactorChanged( double scaleFactor );
 
+    /**
+     * Returns TRUE if the 3D axis controller will handle the specified \a event,
```

Comment:
Can you also adapt the docstring to specify that the event gets handled and return value is whether it consumes the event or not?

Reply:
done!